en/mkdocs.yml

```diff
@@ -102,6 +102,8 @@ nav:
       - File Integration:
         - File Integration With Directory Service: integration-guides/file-integration/file-integration-with-directory-service.md
       - Use Cases:
+        - Integration Service As a Managed API:
+          - How to Expose an Integration Service as a Managed API: integration-guides/usecases/integration-service-as-a-managed-api/how-to-expose-an-integration-service-as-a-managed-api.md
```

Comment:
Since this is one usecase, no need to have sub section, add it directly under `Use Cases`